### PR TITLE
AdminUI - Add Administer Scheduled Reminders page

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Add new screen to the Admin UI Extension.

Before
----------------------------------------
image

After
----------------------------------------
image

Technical Details
----------------------------------------
Added a lot of dynamic metadata so that options appeared correctly for a given type. Dynamic option lists are great; it even pluralizes the units correctly ("4 Days" vs "1 Day").

### CODE REVIEW

**Comment by aydun** [comment]
Test this please

File: ext/civicrm_admin_ui/managed/SavedSearch_Administer_Scheduled_Reminders.mgd.php [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+              'sortable' => TRUE,
+              'rewrite' => '[mapping_id:label] - [entity_value:label]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'absolute_date',
+              'dataType' => 'Date',
+              'label' => E::ts('When'),
+              'sortable' => TRUE,
+              'empty_value' => '[start_action_offset] [start_action_unit:label] [start_action_condition] [start_action_date:label]',
```
Comment (aydun) [inline ext/civicrm_admin_ui/managed/SavedSearch_Administer_Scheduled_Reminders.mgd.php:94]: That's clever!

**Comment by aydun** [comment]
All seems to work well.  Great to see this pushing some boundaries!

**Comment by colemanw** [comment]
Thanks for the review @aydun

**Pull request merged**